openapi.yaml

```diff
@@ -139,54 +139,72 @@ info:
     
     ### Example request and response
     
     Request:
     
     ```http
-    OPTIONS /api/v1/jobs HTTP/1.1
+    POST /api/v1/jobs HTTP/1.1
     Host: openeo.cloudprovider.com
-    Origin: http://client.org:8080
-    Access-Control-Request-Method: POST 
-    Access-Control-Request-Headers: Authorization, Content-Type
+    Origin: https://client.org:8080
+    Authorization: Bearer basic//ZXhhbXBsZTpleGFtcGxl
     ```
     
     Response:
     
     ```http
-    HTTP/1.1 204 No Content
-    Access-Control-Allow-Origin: http://client.org:8080
-    Access-Control-Allow-Credentials: true
-    Access-Control-Allow-Methods: OPTIONS, GET, POST, PATCH, PUT, DELETE
-    Access-Control-Allow-Headers: Authorization, Content-Type
+    HTTP/1.1 201 Created
+    Access-Control-Allow-Origin: *
+    Access-Control-Expose-Headers: Location, OpenEO-Identifier, OpenEO-Costs, Link
     Content-Type: application/json
+    Location: https://openeo.cloudprovider.com/api/v1/jobs/abc123
+    OpenEO-Identifier: abc123
     ```
     
-    ## CORS headers
+    ## OPTIONS method
     
-    The following headers MUST be included with every response:
+    All endpoints must respond to the `OPTIONS` HTTP method. This is a response for the preflight requests made by web browsers before sending the actual request (e.g. `POST /jobs`). It needs to respond with a status code of `204` and no response body. In addition to the HTTP headers shown in the table above, the following HTTP headers MUST be included with every response to an `OPTIONS` request:
     
     | Name                             | Description                                                  | Example |
     | -------------------------------- | ------------------------------------------------------------ | ------- |
-    | Access-Control-Allow-Origin      | Allowed origin for the request, including protocol, host and port. It is RECOMMENDED to return the value of the request's origin header. If no `Origin` is sent to the back-end CORS headers SHOULD NOT be sent at all. | `http://client.isp.com:80` |
-    | Access-Control-Allow-Credentials | If authorization is implemented by the back-end the value MUST be `true`. | `true` |
-    | Access-Control-Expose-Headers    | Some endpoints send non-safelisted HTTP response headers such as `OpenEO-Identifier` and `OpenEO-Costs`. All headers except `Cache-Control`, `Content-Language`, `Content-Type`, `Expires`, `Last-Modified` and `Pragma` must be listed in this header. Currently, the openEO API requires at least the following headers to be listed: `Location, OpenEO-Identifier, OpenEO-Costs, Link`. | `Location, OpenEO-Identifier, OpenEO-Costs, Link` |
+    | Access-Control-Allow-Headers     | Comma-separated list of HTTP headers allowed to be send. MUST contain at least `Authorization` if any kind of authorization is implemented by the back-end. | `Authorization, Content-Type` |
```

Comment:
> Comma-separated list of HTTP headers allowed to be send.

Comma-separated list of HTTP headers allowed to be sent with the actual (non-preflight) request.

Reply:
It's a bit unclear to me, but should the`Access-Control-Allow-Headers` always be sent, or only if the OPTIONS request has a `Access-Control-Request-Headers` header?

Reply:
👍